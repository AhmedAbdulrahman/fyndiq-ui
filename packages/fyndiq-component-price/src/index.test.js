import React from 'react'
import { shallow } from 'enzyme'

import { Price, OldPrice, CurrentPrice } from './'

describe('Price Component', () => {
  it('should have a default structure', () => {
    expect(shallow(<Price />)).toMatchSnapshot()
  })

  it('should have a emphasize prop', () => {
    expect(shallow(<Price emphasize={false} />)).toMatchSnapshot()
  })
})

describe('OldPrice Component', () => {
  it('should have a default structure', () => {
    expect(shallow(<OldPrice>125 kr</OldPrice>)).toMatchSnapshot()
  })
})

describe('CurrentPrice Component', () => {
  it('should have a default structure', () => {
    expect(shallow(<CurrentPrice>125 kr</CurrentPrice>)).toMatchSnapshot()
  })
})
