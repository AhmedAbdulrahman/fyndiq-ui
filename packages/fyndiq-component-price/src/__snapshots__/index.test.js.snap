// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CurrentPrice Component should have a default structure 1`] = `
<span
  className="currentPrice"
>
  125 kr
</span>
`;

exports[`OldPrice Component should have a default structure 1`] = `
<span
  className="oldPrice"
>
  125 kr
</span>
`;

exports[`Price Component should have a default structure 1`] = `
<span
  className="
      price
      emphasize
    "
/>
`;

exports[`Price Component should have a emphasize prop 1`] = `
<span
  className="
      price
      false
    "
/>
`;
